packages/core/tests/integration/components/chart-series-config-test.ts

```diff
@@ -53,7 +53,7 @@ module('Integration | Component | chart series config', function(hooks) {
 
     assert.deepEqual(
       findAll('.line-chart-config__series-config__item__content').map(el => el.textContent?.trim()),
-      ['Property 1', 'Property 2', 'Property 3'],
+      ['Property 1', 'Property 2', 'Property 3', 'Property 4'],
```

Comment:
Is the extra dimension/series necessary?

Reply:
This is a result of the rework around dimension series charts now always being invalid, so it pulls in the `Property 4` dimension on load since it wasn't filtered out